react-renderer/src/engine/Optimizer.ts

```diff
@@ -243,11 +275,21 @@ export const step = (state: State, steps: number, evaluate = true) => {
         optParams.optStatus.tag = "UnconstrainedConverged";
         memoryManager.deleteExcept([]); // Clear allocated matrix, vector objects in L-BFGS params
         optParams.lbfgsInfo = defaultLbfgsParams;
-        console.log("Unconstrained converged with energy", energyVal, "gradient norm", normGrad);
+        console.log(
```

Comment:
Is it possible to turn off the "automatically put all arguments on a new line" setting? I'm fine with it being used in some files, but prefer my code to be more concise 🙂

Reply:
Yeah, I commented on the `console.log` but I would also like the function calls, JS objects, etc. to be OK to write on one line, rather than spacing out all the arguments. Would that be possible?

Reply:
I'd look at https://github.com/penrose/penrose/blob/57a0e7fe954146761270ff37f1190f34e234cea0/react-renderer/.eslintrc.js and refer to the link above. Not sure what the defaults are, but I'm open to change lint rules if you think it'd be helpful.